### DISCUSSION TITLE
Bluetooth sample don´t work

### DISCUSSION THREAD

**Discussion by joaodiogogithub** [zephyrproject-rtos/zephyr#60592] [Q&A]
Hello, every time that i try to build an Bluetooth sample, an error appear :
![Captura de tela de 2023-07-19 15-00-29](https://github.com/zephyrproject-rtos/zephyr/assets/101523748/af127ac3-9d4e-407f-88ae-2fb98a0e9df8)

**Comment by erwango** [comment]
@Bell might be able to help

**Comment by joaodiogogithub** [comment]
Guys, I achieved to fix this, all this time i did wrong, i try to build the core program on the net board.
